src/components/CommentTile.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react"
+import axios from "axios"
+
+export default function CommentTile ({comment}) {
+    const oldTime = comment.created_at
+    let date = oldTime.substring(0, 10)
+    let time = oldTime.substring(11, 16)
+
+    const [userPicture, setUserPicture] = useState('')
+
+    const usersApi = axios.create({
+        baseURL: 'https://nc-news-p6kr.onrender.com/api/users'
+    })
```

Comment:
would deffo extract this into a api.js file on a more root-level location so you don't have to keep creating base URLs. It feels like it isn't really drying up your code to do it this far down in your app files